### PR TITLE
Added Czech localization

### PR DESCRIPTION
**Pull request by harryheres**
# What it Does
* Adds a new localization option for the Czech language

# How I Tested
* Went through the whole app and checked every string present to the user.

# Notes
* None

# Screenshots
image

image

### CODE REVIEW

**Review by mikaelacaron — CHANGES_REQUESTED** [review]
@HarryHeres can you solve these merge conflicts please? unfortunately it looks like you may have to fix each translation one by one, in some kind of editor, because I'm guessing this wont't build in Xcode, cause it can't parse the Localizable.xcstrings file

**Comment by harryheres** [comment]
@mikaelacaron Conflicts fixed :). This might happen also in the future, since there are many of us working on the translations.

**Comment by mikaelacaron** [comment]
Yeah it's definitely going to happen again, did you solve it by using an editor outside of Xcode?

**Comment by harryheres** [comment]
> Yeah it's definitely going to happen again, did you solve it by using an editor outside of Xcode?

I do all of my other work in Neovim, so fired that up and manually fixed all. I suppose as more languages will get added and get translated, the amount of merge conflicts will eventually lower. But at the moment, we will need to resolve these manually, becase the Localization source file is just a JSON in the end, so after resolving it, I recommend using a JSON analyzer to check the structure. If it is fine, XCode will accept it and build it out.

File: Basic-Car-Maintenance/Shared/Localizable.xcstrings [hunk 1]
```diff
@@ -43,29 +67,48 @@
     },
     "Add Vehicle" : {
       "localizations" : {
-        "de" : {
+        "cs" : {
           "stringUnit" : {
             "state" : "translated",
-            "value" : "Fahrzeug hinzufügen"
+            "value" : "Přidat vozidlo"
```
Comment (mikaelacaron) [inline Basic-Car-Maintenance/Shared/Localizable.xcstrings]: Is this overwriting the German translation?

File: Basic-Car-Maintenance/Shared/Localizable.xcstrings [hunk 2]
```diff
@@ -43,29 +67,48 @@
...
+            "state" : "translated",
+            "value" : "Další poznámky"
+          }
+        },
         "de" : {
           "stringUnit" : {
             "state" : "translated",
-            "value" : "Weitere Notizen"
+            "value" : "Fahrzeug hinzufügen"
           }
```
Comment (mikaelacaron) [inline Basic-Car-Maintenance/Shared/Localizable.xcstrings]: Is this overwriting the German translation?

**Review by mikaelacaron — CHANGES_REQUESTED** [review]
Just two questions!
Yeah this makes sense that there will be merge conflicts, it depends on when someone forks the project and makes the translation

File: Basic-Car-Maintenance/Shared/Localizable.xcstrings (hunk 2 above)
Comment (harrythewise) [reply Basic-Car-Maintenance/Shared/Localizable.xcstrings]: Oh, it seems like I must’ve missed a line! Will fix that tomorrow. Sorry!

File: Basic-Car-Maintenance/Shared/Localizable.xcstrings (hunk 1 above)
Comment (harrythewise) [reply Basic-Car-Maintenance/Shared/Localizable.xcstrings]: Oh, it seems like I must’ve missed a line! Will fix that tomorrow. Sorry!

File: Basic-Car-Maintenance/Shared/Localizable.xcstrings (hunk 2 above)
Comment (harrythewise) [reply Basic-Car-Maintenance/Shared/Localizable.xcstrings]: Fixed.

File: Basic-Car-Maintenance/Shared/Localizable.xcstrings (hunk 1 above)
Comment (harrythewise) [reply Basic-Car-Maintenance/Shared/Localizable.xcstrings]: Fixed.

**Review by mikaelacaron — APPROVED** [review]
Great work! Thanks for solving the merge conflicts!

**Pull request merged**